### ISSUE TITLE
Add support for Refined

### ISSUE THREAD

**Issue by hderms** [andyglow/scala-jsonschema#14]
https://github.com/fthomas/refined

**Comment by andyglow** [comment]
Yeah. That is what I planed to do. Can’t give any estimation now, but that is in my backlog. If you want it is always very welcome for somebody to contribute, though )

**Comment by andyglow** [comment]
Refined support was just added 
Checkout`0.2.5` release.

**Comment by hderms** [comment]
Wow thanks. This library is amazing

**Comment by gordol** [comment]
this doesn't work currently, any idea why it's removed?

**Comment by andyglow** [comment]
> this doesn't work currently, any idea why it's removed?

@gordol could you describe your issue? please include your `build.sbt` and a code snippet that doesn't work. Thanks!

**Comment by gordol** [comment]
The whole refined module add-on is missing from maven. I had to fork this repo, add the refined module to line 280 at the bottom of build.sbt, rebuild locally, and install the library to my local repo. After doing this, everything works...

So, why isn't the refined module enabled?

**Comment by andyglow** [comment]
oops. my bad. hasn't been released accidentally. 
now it's there https://repo1.maven.org/maven2/com/github/andyglow/scala-jsonschema-refined_2.13/
in several hours will be available in maven central
